public/.user.ini

```diff
@@ -1,2 +1,3 @@
-upload_max_filesize = 10M
-memory_limit = 256M
\ No newline at end of file
+upload_max_filesize = 2000M
+memory_limit = 2048M
```

Comment:
Have you been upping the dyno size when testing this setting? Chompy QA is running on a hobby dyno, which [has only 512MB of RAM](https://devcenter.heroku.com/articles/dyno-types).

Reply:
omg I didn't even check that! I didn't up the dyno size - should I do that to standard-2-x?

Reply:
@DFurnes what is the difference between upping this in the code versus on Heroku?

Reply:
> omg I didn't even check that! I didn't up the dyno size - should I do that to `standard-2-x`?

I think that's a good first try. We probably don't want to be paying for performance dynos for this application, so if that doesn't help then we'll have to find solutions elsewhere.

> @DFurnes what is the difference between upping this in the code versus on Heroku?

You're telling PHP how much memory it is _allowed_ to use per-request when setting [`memory_limit`](http://php.net/manual/en/ini.core.php#ini.memory-limit), but there's still a "cap" on how much memory is available (and even if you ask nicely, you can't use more than you have).

Reply:
Yeah, let's give it a try at least!

Reply:
@DFurnes ok! So this time after increasing the dyno size, it got through 54,877 records! What I would love to do is:
- Wipe NS QA tonight/tomorrow am before office hours (optional, I can also just try last 50k records even though there might be some overlap)
- Re-test with 50k records

If all goes well tomorrow, we can keep that dyno for a bit and merge this to master? 
cc @DoSomething/team-bleed

Reply:
Woohoo, that's much nicer! 👏 Totally onboard for a QA reset, just drop a note in #deploys!

Reply:
cool, will do!! :)

Reply:
Just want to give an update since I know @ngjo wants to do some RTV chomping! This has been running since 10:40am. It's only created 4,878 new posts and has only gone through 3,273 records it looks like... I'm not sure why it is so slow. The thor refresh did get kicked off after I started the script started so not sure if that affected anything?

Reply:
@chloealee does clearing the cache work for that? I'm not sure. Also the Northstar QA database has been struggling all day so I think the errors you're seeing are due to that

Reply:
ah thanks for letting me know about that @katiecrane ! I think that _might_ be causing the job to run so slow. Perhaps I'll just keep it going then and continue to keep an eye on it rather than starting from scratch since that likely won't change any behavior if NS QA has been struggling all day. Thanks, Katie!

Reply:
stopped this job (by clearing the cache - thanks @katiecrane !) as it was craaawling and I think we can maybe regroup as a team re: next steps on Monday since some people are out and we probably wouldn't get to RTV imports then anyway! 

At first, It seemed like this job was really heavy for Northstar QA and slowed it down a lot. @katiecrane made a great point that even though we beefed up the queue dyno for Chompy QA, it might have been too much for Northstar QA. 

In more than 24 hours, it seemed to only get through getting the user info. of ~8,800 records.

After stopping this job, Northstar QA's performance didn't improve so perhaps this was just a coincidence and that is why the RTV import was moving so slowly. 

Lets re-group on Monday re: next steps and by then we should know more info. about NS too!

Reply:
hi team! I would like to re-start this on QA and test again but wanted to check here first - is this ok with everyone?

Reply:
@DoSomething/team-bleed looks like this stopped after creating 8,155 new voter reg posts of the 50k record import :-/ It was running for almost exactly 4 hours when it recorded the last thing of from this job in the [logs](https://papertrailapp.com/systems/dosomething-chompy-qa/events?focus=986764510621347841&selected=986764510621347841). I don't see anything in the logs to indicate any errors though. @DFurnes is there a way to see how much memory we used of the 1 GB of RAM in heroku?

Reply:
@chloealee You can look at metrics per formation in [Heroku's metrics tab](https://dashboard.heroku.com/apps/dosomething-chompy-qa/metrics/queue). It looks like we never used above ~200MB of RAM (which makes sense to me now that we've split these jobs out)!

It might be helpful to jump on a call tomorrow or Friday and talk this through to see if there's anything we're missing that could be causing trouble here, but IMO this PR is totally good to merge and deploy.

Reply:
cool - thank you for sharing that @DFurnes ! 

I'd love to jump on a call today if you're free - I don't have any meetings at all today so just lmk when you're free! :) Would love to discuss if there is anything we're missing and if we can't think of anything, what is a safe number of records per file we can tell @ngjo is safe to upload and that we're confident will make through the entire file.